config/default/namespace_delete.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```

Comment:
can we delete all the `system` namespace so that there is no needs for this delete patch?